external/ilasm/ilasm.depproj

```diff
@@ -2,10 +2,7 @@
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
-    <!-- Always restore the x64 assets. These are build-only assets, so they should match the environment of the build. -->
-    <NugetRuntimeIdentifier>$(RuntimeOS)-x64</NugetRuntimeIdentifier>
-    <!-- Tizen does not provide the x64 runtime so replaces it with the ubuntu.14.04-x64 -->
-    <NugetRuntimeIdentifier Condition="'$(RuntimeOS)' == 'tizen.4.0.0'">ubuntu.14.04-x64</NugetRuntimeIdentifier>
+    <NugetRuntimeIdentifier>$(RuntimeRID)</NugetRuntimeIdentifier>
```

Comment:
Based on the change above (with your comment), it looks like it will still be x64. Technically we just need to restore an ILAsm that is compatible with the build environment, so an x86 version would/should work for Windows builds. But it's probably best and safest to keep it using the x64 version.